plugins/inventory/foreman.py

```diff
@@ -238,10 +297,200 @@ def _get_hostvars(self, host, vars_prefix='', omitted_vars=()):
                 hostvars[vars_prefix + k] = v
         return hostvars
 
+    def to_safe(self, word):
```

Comment:
There is a `to_safe_group_name` that we import from `ansible.plugins.inventory` and use in the non-report version, should it be used here too, instead of implementing it ourself?